yt_dlp/downloader/hls.py

```diff
@@ -89,7 +89,7 @@ def real_download(self, filename, info_dict):
                 self.report_warning('[%s] pycryptodome is needed to download this file' % self.FD_NAME)
             fd = FFmpegFD(self.ydl, self.params)
             self.report_warning(
-                '[%s] Detected unsupported features, extraction will be delegated to %s' % fd.get_basename())
+                '[%s] Detected unsupported features, extraction will be delegated to %s' % (self.FD_NAME, fd.get_basename()))
```

Comment:
```suggestion
                '%s detected unsupported features; extraction will be delegated to %s' % (self.FD_NAME, fd.get_basename()))
```

is more in-line with your other changes, no?

Reply:
Yes, it’s fine